go.mod

```diff
@@ -25,23 +25,20 @@ require (
 	github.com/spf13/cobra v1.9.1
 	github.com/stretchr/testify v1.10.0
 	golang.org/x/exp v0.0.0-20250228200357-dead58393ab7
+	golang.org/x/mod v0.24.0
 	golang.org/x/sync v0.12.0
 	golang.org/x/tools v0.31.0
 	gopkg.in/yaml.v2 v2.4.0
 	helm.sh/helm/v3 v3.17.2
-	k8s.io/api v0.32.3
-	k8s.io/apiextensions-apiserver v0.32.2
-	k8s.io/apimachinery v0.32.3
-	k8s.io/apiserver v0.32.3
-	k8s.io/cli-runtime v0.32.3
-	k8s.io/client-go v0.32.3
-	k8s.io/component-base v0.32.3
 	k8s.io/klog/v2 v2.130.1
+	k8s.io/kubernetes v1.32.3
 	k8s.io/utils v0.0.0-20241210054802-24370beab758
 	sigs.k8s.io/controller-runtime v0.20.2
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require k8s.io/kube-openapi v0.0.0-20241105132330-32ad38e42d3f // indirect
+
```

Comment:
Any idea why this is now moved into a separate `require` statement? Before it was in the `require` grouping of all of the indirect dependencies.

Reply:
maybe a bug in the k8smaintainer code? I'll check there

Reply:
refactored k8smaintainer code, see below, the long reply comment, but I don't think we're going to get better control of the file layout, golang.org/x/mod is controlling that.